apps/outline/docker-compose.yml

```diff
@@ -0,0 +1,97 @@
+version: "3.8"
+
+services:
+  outline:
+    container_name: outline
+    image: docker.getoutline.com/outlinewiki/outline:0.75.2
+    restart: unless-stopped
+    environment:
+      - DATABASE_URL=postgres://outline:${OUTLINE_PG_PASSWORD}@postgres:5432/outline
+      - REDIS_URL=redis://redis:6379
+      - SECRET_KEY=${OUTLINE_SECRET_KEY}
+      - UTILS_SECRET=${OUTLINE_UTILS_SECRET}
+      - URL=https://${APP_DOMAIN}
+      - FORCE_HTTPS=false
+      - FILE_STORAGE=local
+      - FILE_STORAGE_UPLOAD_MAX_SIZE=26214400
+      - PGSSLMODE=disable
+      - OIDC_CLIENT_ID=${OUTLINE_OIDC_CLID}
+      - OIDC_CLIENT_SECRET=${OUTLINE_OIDC_CLKEY}
+      - OIDC_AUTH_URI=https://${APP_DOMAIN}/uc/oauth/authorize/
+      - OIDC_TOKEN_URI=https://${APP_DOMAIN}/uc/oauth/token/
+      - OIDC_USERINFO_URI=https://${APP_DOMAIN}/uc/oauth/userinfo/
+      - OIDC_USERNAME_CLAIM=preferred_username
+      - OIDC_DISPLAY_NAME=OpenID
+      - OIDC_SCOPES=openid profile email
+    volumes:
+      - ${APP_DATA_DIR}/data/storage:/var/lib/outline/data 
+    depends_on:
+      - postgres
+      - redis
```

Comment:
You forgot to change this and the app isn't working. Please make sure you changed in everywhere (including nginx conf files etc)